packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -48,7 +48,8 @@ class WebpackCLI {
 
     async resolveArgs(args, configOptions = {}) {
         // Since color flag has a default value, when there are no other args then exit
-        if (Object.keys(args).length === 1) return {};
+        // eslint-disable-next-line no-prototype-builtins
+        if (Object.keys(args).length === 1 && args.hasOwnProperty('color')) return {};
```

Comment:
Why we have `color` property here?

Reply:
because color is default in args, we only need to skip it in that case

Reply:
in tests we're passing properties which are not color

Reply:
hm, why we have `color` in args?

Reply:
I think default is assigned there, will take a look and remove next

Reply:
Yep, let's do it in the new PR

Reply:
WIP already on color https://github.com/webpack/webpack-cli/pull/2042#discussion_r518730261